Geometry_Engine/Compute/DistributeOutlines.cs

```diff
@@ -35,9 +37,18 @@ public static partial class Compute
         /**** Public Methods                            ****/
         /***************************************************/
 
-        [DeprecatedAttribute("Moved to Common_Engine")]
+        [Description("Distributes outlines into lists representing containing outlines and the outlines that they each contain.")]
+        [Input("outlines", "The outlines to sort.")]
+        [Input("tolerance", "The tolerance to apply when detecting if outlines are closed and contained.")]
+        [Output("distributedOutlines", "Lists representing each set of outlines. For each list, the first outline is the containing outline and the remainder are the outlines it contains.")]
         public static List<List<Polyline>> DistributeOutlines(this List<Polyline> outlines, double tolerance = Tolerance.Distance)
         {
+            if (outlines == null || outlines.Where(x => x != null).Count() == 0)
```

Comment:
Personally I would use different syntax, but it is rather a matter of preference (negligible difference in performance in favour of my proposal I think, one iteration over the collection avoided).
```suggestion
            if (outlines == null || outlines.All(x => x == null))
```

Reply:
will leave as is as I'm with you, its an improvement but a negligible one.